website/content/en/preview/concepts/disruption.md

```diff
@@ -177,6 +181,63 @@ To enable interruption handling, configure the `--interruption-queue-name` CLI a
 
 ## Controls
 
+### Disruption Budgets
+
+You can rate limit Karpenter's disruption through the `NodePool.Spec.Disruption.Budgets`. If undefined, Karpenter will default to one budget with `nodes: 10%`. Budgets will take into account nodes that are being deleted for any reason, and will only block Karpenter from disrupting nodes voluntarily through expiration, drift, emptiness, and consolidation.
+
+#### budget.Duration + budget.Schedule
+Duration and Schedule can only be defined together. When omitted, the budget is always active. When defined, the schedule determines when a budget is active, and the duration determines how long from that point the budget is active. 
+
+##### Schedule
+Schedule is a cronjob schedule. Generally, the cron syntax is five space-delimited values with options below. 
+Follow the [Kubernetes documentation](https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/#writing-a-cronjob-spec) for how to follow the cron syntax. 
+
+```bash
+# ┌───────────── minute (0 - 59)
+# │ ┌───────────── hour (0 - 23)
+# │ │ ┌───────────── day of the month (1 - 31)
+# │ │ │ ┌───────────── month (1 - 12)
+# │ │ │ │ ┌───────────── day of the week (0 - 6) (Sunday to Saturday;
+# │ │ │ │ │                                   7 is also Sunday on some systems)
+# │ │ │ │ │                                   OR sun, mon, tue, wed, thu, fri, sat
+# │ │ │ │ │
+# * * * * *
+```
+
+{{% alert title="Note" color="primary" %}}
+Timezones are not supported. Most images default to UTC, but it is best practice to ensure this is the case when considering how to define your budgets. 
+{{% /alert %}}
+
+##### Duration
+Duration is a metav1.Duration, which allows compound durations with minutes and hours values such as `10h5m` or `30m` or `160h`. Since cron syntax does not accept denominations smaller than minutes, users can only define minutes or hours. 
```

Comment:
To me, this is fairly standard syntax to me. How much do we need to explain how to write this out? For the other fields that use duration, it's assumed so I wonder if we can shorten this

Reply:
I cut down a little bit, but I was thinking two lines couldn't hurt for users coming straight to this. Thoughts?